pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -1,6 +1,9 @@
 description: |
   AP plots and metrics to assess CcdVisit quality in Prompt Processing
   Additional AP metrics may be written in apDetectorVisitQualityExtended
+parameters:
+  coaddName: goodSeeing
+  fakesType: ''
```

Comment:
Does this need a comment saying that the parameter names are shared with `ApPipe` (I assume that's the intent)?